tests/Functional/app/ORM/config.yml

```diff
@@ -16,8 +16,8 @@ services:
         alias: fos_elastica.indexable
     indexable_service:
         class: FOS\ElasticaBundle\Tests\Functional\app\ORM\IndexableService
-    fos_elastica.manager.orm:
-        class: FOS\ElasticaBundle\Doctrine\RepositoryManager
+    test_alias.fos_elastica.manager.orm:
+        alias: fos_elastica.manager.orm
```

Comment:
Should better work now ;)

Reply:
Yes tests are finally passing :P

Reply:
Maybe we can set only alias to public true and avoid setting public the indexable_service?

Reply:
Like this: https://github.com/FriendsOfSymfony/FOSElasticaBundle/pull/1664 ?

Reply:
Yes but sure the indexable_service can be private ;)

Reply:
Well, it was private before so it should still work.